src/browzine-summon-adapter.js

```diff
@@ -1,30 +1,52 @@
-// Begin BrowZine-Summon Integration Code
-angular.module("summonApp.directives")
-.constant("api", "https://api.thirdiron.com/public/v1/libraries/XXX")
-.constant("apiKey", "ENTER API KEY")
-.constant("bookIcon", "https://s3.amazonaws.com/thirdiron-assets/images/integrations/browzine_open_book_icon.png")
-.directive("documentSummary", ["$http", "$sce", "api", "apiKey", "bookIcon", function(http, sce, api, apiKey, bookIcon) {
+browzine.search = (function() {
+  var api = browzine.api;
+  var apiKey = browzine.apiKey;
+
   function isArticle(data) {
-    if(typeof data.document !== "undefined" && data.document !== null) {
-      return data.document.content_type.trim() === "Journal Article";
-    } else {
-      return data.type === "articles";
+    var result = false;
+
+    if(data.document) {
+      if(data.document.content_type) {
+        var contentType = data.document.content_type.trim();
+        result = (contentType === "Journal Article");
+      }
+    }
+
+    if(data.type) {
+      result = (data.type === "articles");
     }
+
+    return result;
   };
 
   function isJournal(data) {
-    if(typeof data.document !== "undefined" && data.document !== null) {
-      return data.document.content_type.trim() === "Journal";
-    } else {
-      return data.type === "journals";
+    var result = false;
+
+    if(data.document) {
+      if(data.document.content_type) {
+        var contentType = data.document.content_type.trim();
+        result = (contentType === "Journal" || contentType === "eJournal");
+      }
+    }
+
+    if(data.type) {
+      result = (data.type === "journals");
     }
+
+    return result;
   };
 
   function getIssn(scope) {
     var issn = "";
 
-    if(typeof scope.document.issns !== "undefined" && scope.document.issns !== null) {
-      issn = scope.document.issns[0].trim().replace('-', '');
+    if(scope.document) {
+      if(scope.document.issns) {
+        issn = scope.document.issns[0].trim().replace('-', '');
+      }
+
+      if(scope.document.eissns && issn === "") {
```

Comment:
Total nit here, but do we want to use `''` here and on line 40, since we also use single quotes in the `replace` calls near here?

Looks like we're using a mix of double and single quotes throughout the repo - any preference on one or the other?

Reply:
Good catch. You should see double quotes mainly, then single quotes when specifying template properties:
```
"<div class='browzine'>"
```